src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/ValidationContext.cs

```diff
@@ -60,11 +60,6 @@ public class ValidationContext
             CancellationToken token,
             ILogger<ValidationContext> logger)
         {
-            if (entries == null)
```

Comment:
We should leave a comment here about what cases entries will be non and non-null

Reply:
Added to the docs of `ValidationContext.Entries` itself.